ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java

```diff
@@ -3250,4 +3250,8 @@
     @LogMessage(level = WARN)
     @Message(id = 529, value = "Failed to retrieve info from database for timer: %s")
     void failedToRetrieveTimerInfo(final TimerImpl timer, @Cause Exception e);
+
+    @Message(id = 530, value = "The deployment is configured to use legacy security which is no longer supported.")
+    IllegalStateException legacySecurityUnsupported();
+
```

Comment:
is it possible to include some context info like the security domain name used?

Reply:
Thanks @chengfang I will have a look if I can add this.